commit 8b5628ab83b671f96ac9f174c1bd51c92589fc82
Merge: a8f3740feb12 fe7dc7202d7d
Author: Linus Torvalds <torvalds@linux-foundation.org>
Date:   Thu Feb 21 15:40:16 2013 -0800

    Merge tag 'virt' of git://git.kernel.org/pub/scm/linux/kernel/git/arm/arm-soc
    
    Pull ARM virtualization changes:
     "This contains parts of the ARM KVM support that have dependencies on
      other patches merged through the arm-soc tree.  In combination with
      patches coming through Russell's tree, this will finally add full
      support for the kernel based virtual machine on ARM, which has been
      awaited for some time now.
    
      Further, we now have a separate platform for virtual machines and qemu
      booting that is used by both Xen and KVM, separating these from the
      Versatile Express reference implementation.  Obviously, this new
      platform is multiplatform capable so it can be combined with existing
      machines in the same kernel."
    
    * tag 'virt' of git://git.kernel.org/pub/scm/linux/kernel/git/arm/arm-soc: (38 commits)
      ARM: arch_timer: include linux/errno.h
      arm: arch_timer: add missing inline in stub function
      ARM: KVM: arch_timers: Wire the init code and config option
      ARM: KVM: arch_timers: Add timer world switch
      ARM: KVM: arch_timers: Add guest timer core support
      ARM: KVM: Add VGIC configuration option
      ARM: KVM: VGIC initialisation code
      ARM: KVM: VGIC control interface world switch
      ARM: KVM: VGIC interrupt injection
      ARM: KVM: vgic: retire queued, disabled interrupts
      ARM: KVM: VGIC virtual CPU interface management
      ARM: KVM: VGIC distributor handling
      ARM: KVM: VGIC accept vcpu and dist base addresses from user space
      ARM: KVM: Initial VGIC infrastructure code
      ARM: KVM: Keep track of currently running vcpus
      KVM: ARM: Introduce KVM_ARM_SET_DEVICE_ADDR ioctl
      ARM: gic: add __ASSEMBLY__ guard to C definitions
      ARM: gic: define GICH offsets for VGIC support
      ARM: gic: add missing distributor defintions
      ARM: mach-virt: fixup machine descriptor after removal of sys_timer
      ...

diff --cc drivers/clocksource/Makefile
index 596c45c2f192,49271105339f..7d671b85a98e
--- a/drivers/clocksource/Makefile
+++ b/drivers/clocksource/Makefile
@@@ -17,7 -17,6 +17,7 @@@ obj-$(CONFIG_CLKSRC_DBX500_PRCMU)	+= cl
  obj-$(CONFIG_ARMADA_370_XP_TIMER)	+= time-armada-370-xp.o
  obj-$(CONFIG_ARCH_BCM2835)	+= bcm2835_timer.o
  obj-$(CONFIG_SUNXI_TIMER)	+= sunxi_timer.o
 +obj-$(CONFIG_ARCH_TEGRA)	+= tegra20_timer.o
  obj-$(CONFIG_VT8500_TIMER)	+= vt8500_timer.o
  
- obj-$(CONFIG_CLKSRC_ARM_GENERIC)	+= arm_generic.o
+ obj-$(CONFIG_ARM_ARCH_TIMER)		+= arm_arch_timer.o